database/migrations/2020_06_16_215943_create_mentees_table.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateMenteesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('mentees', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->timestamps();
+            $table->text('currentstatus')->nullable();
+            $table->text('previousexp')->nullable();
+            $table->char('mentortype', 3)->default('No')->nullable();
+            $table->text('timeframe')->nullable();
+            $table->text('suitabletime')->nullable();
+            $table->text('extrainfo')->nullable();
+            $table->char('status', 30)->default('Pending');
```

Comment:
For status type fields (e.g. any field that can have one of a fixed set of values) I've found that using PHP enums makes a big difference to how easy those values are to work with.  I use this package:

https://github.com/BenSampo/laravel-enum

Happy to talk you though usage if you need any pointers beyond what's in the readme